_posts/2021-03-15-Moving-Monsters.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: Moving Monsters 
+description: 
+summary: 
+tags:
+---Today, we started off with typing club once again. Then, Kajari instructed us to work on our game that we have been working on for the past weeks. Today, we had to place both monsters on each side of the top of the canvas. We had to place monster one on the top left side whose coordinates are 0,0. This is because it is the top of the canvas and the coordinates have not progressed which is why it is 0,0. Now, for monster number 2, the coordinates are 448,0. This is because the width stays the same while the height increases, because we want monster 2 to the right side of the canvas. Below I have included a sketch of what kajari provided for me as a reference.
```

Comment:
Move `Today, we...`  to a new line

Reply:
This sentence doesn't mean anything. What are you trying to say? `This is because it is the top of the canvas and the coordinates have not progressed which is why it is 0,0.`

Reply:
This sentence is not true and also doesn't mean anything: `This is because the width stays the same while the height increases,`. What are you trying to say? Are you trying yo say that you are keeping the position along the y axis the same (where it is on the screen when moving to going up and down) and move x (where it is on the screen when moving left and right?) to be the same as the width of the canvas?